apis/cc-infix-en_GB/atrium-api-cc-infix-en_GB-common/src/main/kotlin/ch/tutteli/atrium/api/cc/infix/en_GB/iterableAssertions.kt

```diff
@@ -46,6 +46,15 @@ infix fun <E, T : Iterable<E>> Assert<T>.notTo(@Suppress("UNUSED_PARAMETER") con
  * @return This plant to support a fluent API.
  * @throws AssertionError Might throw an [AssertionError] if the assertion made is not correct.
  */
+@Deprecated(
+    "Switch from Assert to Expect; will be removed with 1.0.0 -- see https://github.com/robstoll/atrium/releases/tag/v0.9.0#migration for migration hints and scripts.",
```

Comment:
don't forget `to contain` and `notTo contain` above

Reply:
I'm not sure about the replacement of to contain and notTo contain.
Are there any examples?

Reply:
you can always find the correspondence in the new infix API. The structure of the files should be similar. i.e. have a lookt at iterableAssertions.kt in api-infix

Reply:
Hmm. 
Why I can't find method `to` in `ch.tutteli.atrium.api.cc.infix.en_GB.to` for replacement

Reply:
It's again the same as for CharSequence, it was renamed and is called `contains(o)` instead of `to contain`.
I guess I was not clear enough what I meant with `same structure`. I try to explain it better. If you deprecate `to contain` in api-cc-infix which is the first thing within iterableAssertions.kt then most likely the same functionality is also the first thing you will see in api-infix. I hope you get what I mean